### DISCUSSION TITLE
Can not get it to load

### DISCUSSION THREAD

**Discussion by crosenblum** [nicolargo/glances#2752] [General]
**Describe the bug**
At first the major issue was where to create the initial configuration file, what folder path, and what is the name of that file.

According to docker configuration, you mention "glance.yml" which according to the documentation is a file, with the current configuration.

But docker set it up as a folder, so I sshed into that folder, created a file called "glance.conf" and still it will not work.

I just want to be clear what folder, to put the configuration file in, and what is the name of the file that is that configuration file.

**To Reproduce**
1. Install Glance via docker compose either raw or via portainer stack creator.

services:
  glance:
    image: glanceapp/glance
    volumes:
      - ./glance.yml:/app/glance.yml
      - /etc/timezone:/etc/timezone:ro
      - /etc/localtime:/etc/localtime:ro
    ports:
      - 8080:8080
    restart: unless-stopped

All as mentioned on here:

https://github.com/glanceapp/glance?tab=readme-ov-file#installation

2. Create the stack, it creates the container
3. Then it creates this repeating error:

"failed parsing config file: read glance.yml: is a directory"

**Expected behavior**
If there is no default configuration file whether a default minimal version, to make sure the install went correctly, that would be useful, then I can find the file, make any changes, and start using glance.

**Screenshots**
https://imgur.com/a/TPqO4ac

**Environement (please complete the following information)**
 - Operating System: DietPi v9.3
 - Glances & psutil versions: Command does not work
 - How do you install Glances Docker/Portainer/Docker Compose

**Comment by nicolargo** [comment]
You are looking for Glance (without a 's'), not Glances (with a 's'). It is two different projects:

**Reply by crosenblum** [reply]
My bad, I apologize, please remove.
